contracts/Oracle/Calculations/Curve.sol

```diff
@@ -74,24 +74,35 @@ interface ICalculationsChainlink {
     function oracleNamehashes(address) external view returns (bytes32);
 }
 
+interface ICurveRegistryOverrides {
+    function poolByLp(address) external view returns (address);
+}
+
 contract CalculationsCurve is Ownable {
     address public yearnAddressesProviderAddress;
     address public curveAddressesProviderAddress;
+    address public curveRegistryOverridesAddress;
     IYearnAddressesProvider internal yearnAddressesProvider;
     ICurveAddressesProvider internal curveAddressesProvider;
+    ICurveRegistryOverrides internal curveRegistryOverrides;
 
     constructor(
         address _yearnAddressesProviderAddress,
-        address _curveAddressesProviderAddress
+        address _curveAddressesProviderAddress,
+        address _curveRegistryOverridesAddress
```

Comment:
CurveRegistryOverrides from the YearnAddressesProvider?

Reply:
Yeah, I guess it'd be good in case we need to change the code in the overrides contract and so we won't have to deploy the curve calculations contract again

Reply:
yearn addresses provider now contains the mapping for curve override via "CURVE_REGISTRY_OVERRIDES"